lib/nncp-l3/ocp_node_template.yaml

```diff
@@ -34,15 +34,14 @@ spec:
         state: up
         type: linux-bridge
         mtu: 1500
-        ipv4:
-          enabled: false
       - description: ctlplane interface
         name: _replaced_
         state: up
         type: ethernet
         mtu: 1500
         ipv4:
-          enabled: false
+          enabled: true
+          dhcp: true
```

Comment:
Has this been tested to see how other DTs that use this will be affected?  Or is BGP currently the only kind of DT that depends on this?

Reply:
Afaik only bgp is using this. It's probably not a hard requirement to have dhcp on the ctlplane nic, just needed because of how ci-fmw does stuff (or better because of how we bend it to do what we want).

Reply:
It looks like only the BGP DTs use this file so I think it's safe for non-BGP DTs.

In more detail, `lib/nncp-l3/ocp_node_template.yaml` is changed. There are 3 possible ocp_node_template.yaml files used by the DTs/VAs.
```
[fultonj@runcible architecture{l3xl}]$ find . -name ocp_node_template.yaml
./lib/nncp/ocp_node_template.yaml
./lib/nncp_ipv6/ocp_node_template.yaml
./lib/nncp-l3/ocp_node_template.yaml
[fultonj@runcible architecture{l3xl}]$ 
```
Which DTs/VAs use nncp-l3?
```
[fultonj@runcible architecture{l3xl}]$ grep -R nncp-l3
examples/dt/bgp_dt01/control-plane/nncp/kustomization.yaml:  - ../../../../../lib/nncp-l3
examples/dt/bgp-l3-xl/control-plane/nncp/kustomization.yaml:  - ../../../../../lib/nncp-l3
[fultonj@runcible architecture{l3xl}]$ 
```